Environment/Environment.tex

```diff
@@ -1550,11 +1550,12 @@ \section{Finding methods}
 \begin{ExecuteSmalltalkScript}
 SBEScreenshotRecorder writeTo: './figures/methodNamesRandom.png' building: [:helper |
 	| methodList |
-	(MessageNames methodBrowserSearchingFor: 'random') openInWorld.
+	(MessageNames methodBrowserSearchingFor: '*random*point*') openInWorld.
 	helper scaleWindow: helper foregroundWindow.
 	helper
-		selectAnyListIn: helper foregroundWindow at: 'atRandom:';
-		selectAnyListIn: helper foregroundWindow at: 'Integer*'.
+		selectAnyListIn: helper foregroundWindow at: 'randomPoint:';
+		selectAnyListIn: helper foregroundWindow at: 'Rectangle*'.
+	self flag: #discuss. "ct: Which example is better, with or without wildcards?"

```

Comment:
@codeZeilen Wdyt, should the screenshot show a simple query ("random") like before or one that illustrates the possibility to use wildcads?

Reply:
With wildcards sounds like a good idea :) The basic case is obvious, I think.

Reply:
```suggestion
```